city_scrapers/spiders/chi_ssa_8.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+import re
+import unicodedata
+from calendar import day_name, month_name
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from w3lib.html import remove_tags
+
+
+class ChiSsa8Spider(CityScrapersSpider):
+    name = "chi_ssa_8"
+    agency = "Chicago Special Service Area #8 Lakeview East"
+    timezone = "America/Chicago"
+    start_urls = ["https://lakevieweast.com/ssa-8/"]
+
+    months = {m.lower() for m in month_name[1:]}
+    days = {d.lower() for d in day_name[1:]}
+
+    def parse(self, response):
+        for item in response.css(".post-content ul"):
+            bodyText = item.extract()
+            if "Location" in bodyText:
```

Comment:
We might not be able to count on "Location" always being in the string, so it might be better to check based off of whether a year string can be found